### ISSUE TITLE
Test Phrase Searching for IIIF Content Search Service

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#5897]
# Description
Implementing support for phrase searching in the IIIF Content Search service. Phrase searching is implicit, and users do not need to use quotes in the text field of the search within component on the media object page to perform explicit phrase searching.

Solr queries may need to change; search will split terms on whitespace and search for individual terms in the transcript by default. Phrase searching could be possible using the highlighting feature in Solr, but not with term counts used in normal searching.

## Example Item
https://avalon-dev.dlib.indiana.edu/media_objects/x920fw86z/manifest.json

# Done Looks Like
  - [x] Punctuation and other special characters are handled properly in the search query and result without anything breaking
  - (rsolr has a sanitizing function already being used, quotes getting stripped out)
  - [x] Text input doesn't break functionality

**Comment by joncameron** [comment]
Default phrase searching has already been set as the default, and this should already be working in Avalon. Phrase searching is working in the transcript component.

**Comment by elynema** [comment]
I tested some examples from the transcript for the first section: https://avalon-dev.dlib.indiana.edu/media_objects/gf06g267d

- it's - worked properly
- Dr. Posin's Giants - worked properly
- television. ? - worked properly, although the display code didn't highlight the matching text properly in the transcript
- [THEME MUSIC] - this search did not find the matches until I dropped the brackets
- --Television  - this search did not find the match until I dropped the leading dashes
- J-U-N-G - this search did not find the any matches, although there is one in the transcript

Note that if I enter quotation marks into the Transcript search box with the intention of forcing a phrase search, the quotation marks mean that the search ends up getting 0 results.
